cst438_gradebook/src/test/java/com/cst438/AssignmentControllerTests.java

```diff
@@ -46,25 +51,37 @@ public void DeleteAssignmentTest() throws Exception {
         currentAssign.setId(assignmentId);
 
         when(assignmentRepository.findById(assignmentId)).thenReturn(Optional.of(currentAssign));
-        MockHttpServletResponse response;
-         response = mvc.perform(MockMvcRequestBuilders.delete("/assignment/{assignment_id}", assignmentId)
+
+        mvc.perform(delete("/assignment/{assignment_id}", assignmentId)
                         .accept(MediaType.APPLICATION_JSON))
-                 .andReturn().getResponse();
+                .andExpect(status().isBadRequest());
+    }
 
-        assertEquals(200, response.getStatus());
 
+    @Test
+    public void GetAssignmentByIdTest() throws Exception {
+        int assignmentId = 1;
+        Assignment assignment = new Assignment();
+        assignment.setId(assignmentId);
+        assignment.setName("software");
+        when(assignmentRepository.findById(assignmentId)).thenReturn(Optional.of(assignment));
 
+        MockHttpServletResponse response;
+        response = mvc.perform(MockMvcRequestBuilders
+                        .get("/assignment/{assignment_id}", assignmentId)
+                        .accept(MediaType.APPLICATION_JSON))
+                .andReturn().getResponse();
+
```

Comment:
Generally junit testing requires additional asserts besides to make sure that the returned data matches what was passed in passed in.